timedctl/timedctl.py

```diff
@@ -458,12 +458,20 @@ def edit_report(self, date):
             error_handler("ERR_REPORT_UPDATE_FAILED")
         msg("Report updated successfully")
 
-    def start_activity(self, comment, customer, project, task, show_archived):
+    def start_activity(self, comment, customer, project, task, show_archived, start):
         """Start recording activity."""
+        if start:
+            # handle invalid input
+            if not re.match(r"^(\d{2}:)?\d{2}:\d{2}$", start):
```

Comment:
consider moving this validation logic to a proper argparse/input validation stage?